src/components/Customer.js

```diff
@@ -1,27 +1,70 @@
 import React, { Component } from "react";
-import { Link, Route } from "react-router-dom";
 import { Table } from "react-bootstrap";
 import { Button, ButtonToolbar } from "react-bootstrap";
 import { AddCustomer } from "./AddCustomer";
 import { EditCustomer } from "./EditCustomer";
-import { BrowserRouter as Router } from "react-router-dom";
-import { FbAuth } from "./FbAuth";
+const Api_Key = "663f670dd8fbc561e29b49e05624dfe3";
 
 export class Customer extends Component {
   constructor(props) {
     super(props);
-    this.state = { customer: [], addModalShow: false, editModalShow: false };
+    this.state = {
+      customer: [],
+      currentWeather: [],
+      addModalShow: false,
+      editModalShow: false,
+      city: undefined,
+      country: undefined,
+      icon: undefined,
+      main: undefined,
+      celsius: undefined,
+      temp_max: undefined,
+      temp_min: undefined,
+      description: ""
+    };
   }
+
   componentDidMount() {
     this.refreshList();
   }
+
+  getWeather = async (city, country, id, name, age, address) => {
+    if (city && country) {
+      const api_call = await fetch(
+        `http://api.openweathermap.org/data/2.5/weather?q=${city},${country}&appid=${Api_Key}`
+      );
+      const response = await api_call.json();
+      console.log(response);
```

Comment:
dont leave unnecessary console.log statements in your code. Should be used only for development